src/Parser.js

```diff
@@ -743,14 +743,25 @@ export default class Parser {
         if (!res) {
             return null;
         }
-        const match = (/^(#[a-f0-9]{3}|#[a-f0-9]{6}|[a-z]+)$/i).exec(res.text);
+        const match = (/^(#[a-f0-9]{3}|#?[a-f0-9]{6}|[a-z]+)$/i).exec(res.text);
         if (!match) {
             throw new ParseError("Invalid color: '" + res.text + "'", res);
         }
+        let color = match[0];
+        if (color.charAt(0) !== "#" && color.length === 6) {
+           // Check for HTML predefined color words that are 6 letters long.
+           // If not predefined, allow a 6-digit color specification without
+           // a leading #, as in the xcolor package's HTML color model.
+           const colorWordRegEx = RegExp("^(silver|maroon|purple|yellow|orange" +
+               "|bisque|indigo|orchid|salmon|sienna|tomato|violet)$",'i');
```

Comment:
Since we're passing through strings whose length aren't 6 maybe we check that `color` matches a hex 6-tuple that way we're not prefix things like `cherry`.  What's LaTeX's behaviour if it gets a named color that isn't defined?  Also, you said this PR was minimal so may we can deal with those situations in subsequent PR.

Reply:
Every supported 6-letter color names seem to be out of hexadecimal digits (`[0-9a-f]`). I think it'd be better to test for `[0-9a-f]{6}`. Also, why not support 3-digit color codes?

Reply:
>What's LaTeX's behaviour if it gets a named color that isn't defined? 

There is an error message such as: `!Package xcolor Error: Undefined color 'cherry'.` quicklatex.com will still render a black expression if the `Report LaTeX errors` box is unchecked. So the error behavior is somewhat similar to KaTeX. That is, KaTeX also will render a black expression.

> maybe we check that color matches a hex 6-tuple

I'll see if that can be done fairly easily.